bin/key_server.js

```diff
@@ -22,9 +22,16 @@ function main() {
   var Token = require('../tokens')(log, config.tokenLifetimes)
   var Password = require('../crypto/password')(log, config)
 
-  var CC = require('compute-cluster')
-  var signer = new CC({ module: __dirname + '/signer.js' })
-  signer.on('error', function () {}) // don't die
```

Comment:
I'm unsure about the performance implications of going from compute-cluster to hashing-in-the-event-loop.  We should try to measure how this changes things before pushing to prod.

You also raised some security concerns re: having the keys directly in memory of the web processes.

How terrible would it be to keep computer-cluster here while using the new jwtool code?